### ISSUE TITLE
Payment method billing information missing in Stripe dashboard in case of google pay.

### ISSUE THREAD

**Issue by jariwalabhavesh** [tipsi/tipsi-stripe#444]
## Before I have submitted the issue 
[X] I have read an [installation guide](https://tipsi.github.io/tipsi-stripe/docs/installation.html)  
[X] I know that for an iOS I need to install pods because I've read the [installation guide](https://tipsi.github.io/tipsi-stripe/docs/installation.html#cocoapods-ios)  
[X] I have read a [linking guide](https://tipsi.github.io/tipsi-stripe/docs/linking.html) and checked that everything is OK like in [manual linking guide](https://tipsi.github.io/tipsi-stripe/docs/linking.html#manual)  
[X] I know that before using `tipsi-stripe` I need to set options for my app as described in [usage guide](https://tipsi.github.io/tipsi-stripe/docs/usage.html)  

## The problem
Google pay transaction has missing details on stripe transaction. I had passed the billing address and name  while generating a payment token using google pay. Still, it is not showing in Name and Billing address in stripe transaction detail in stripe portal.  It is working fine with apple pay. The only issue with google pay.

Google Pay is accepting billing address while payment like. name, address, city, state, country still after deducting charges it is not reflecting to stripe protal.

## Environment
* tipsi-stripe version: 7.4.0
* OS version: Oreo and below (I haven't get chance to test in pie)
* React-Native version:  0.55.1
* com.google.firebase:firebase-core version: 15.0.2 
* com.google.android.gms:play-services-base version: 12.0.1

For `Android`, please provide the following sections from `android/app/build.gradle`:
* android.compileSdkVersion: 27
* android.buildToolsVersion: 27.0.3
* android.defaultConfig.minSdkVersion: 16
* android.defaultConfig.targetSdkVersion: 26
* android.defaultConfig.multiDexEnabled (if exists): true

## Code To Reproduce Issue (Good To Have)
No any specific use case to regenerate this issue i had just pass **billing_address_required:true** in options. here is code.

``` 
 const lineItems = [{
      currency_code: 'USD',
      description: 'Donation',
      total_price: this.donationAmount,
      unit_price: this.donationAmount,
      quantity: '1'
    }]

    const options = {
      total_price: this.donationAmount,
      currency_code: 'USD',
      requiredBillingAddressFields: ['all'],
      billing_address_required: true,
      shipping_address_required: false,
      line_items: lineItems
    }

    stripe.paymentRequestWithAndroidPay(options).then((response) => {
      // reponse will contains payment token for charge. Using that token we can deduct charge.
    }).catch((error) => {
      // error if token generation failed
    })

```

Here you can see in the above code that there is **billing_address_required** is true. So at the time of payment using Google pay it is asking for name and address details. And then return payment token.

Here is google pay response screenshot that I am receiving as a response in  **stripe.paymentRequestWithAndroidPay** method
![googlepay](https://user-images.githubusercontent.com/10993666/53632869-01f44200-3c3c-11e9-945a-4aca390854fb.png)

Here is a screenshot from the stripe portal which shows the same transaction payment method information.
![dashboard](https://user-images.githubusercontent.com/10993666/53632929-24865b00-3c3c-11e9-8e12-ba716846dc5f.png)

**Comment by jariwalabhavesh** [comment]
@cybergrind : Any thought on this

**Comment by cybergrind** [comment]
It should return billing information with token. I'm not sure that it should be visible on stripe dashboard if your backend doesn't send it. Probably you need to save card with billing information first and then use this card when creating charges.

**Comment by jariwalabhavesh** [comment]
@cybergrind: Thanks for the response, Backend is only deducting charge based in received token from Google Pay, Apple Pay, and Credit card. In the case of Apple Pay, it is receiving on a dashboard board. But I had observed a difference between response generated from generating token. I had already attached response screenshot in issue description. 
Here is a response from Apple Pay which I am getting from stripe.paymentRequestWithApplePay
![apple_pay_reponse](https://user-images.githubusercontent.com/10993666/54031970-4cdefe00-41d6-11e9-88f8-646dbb5e6c05.png)

See the card object in both images(one attached in issue and second attached in comment). Billing information missing in case of Google pay.
